debian/rules

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
```

Comment:
There's some trailing newlines in various files, including this one.. can we remove them?